docs/cli.md

```diff
@@ -361,14 +363,13 @@ Note that the type of the file is now `stripped.Catalog` and it no longer contai
 
 ## `trestle partial-object-validate`
 
-OSCAL objects are extremely large. Some systems may only be able to produce partial OSCAL objects. For example
-the tanium-to-oscal task produces the `results` attribute of an `assessment-results` object.
+OSCAL objects are extremely large. Some systems may only be able to produce partial OSCAL objects. For example, the tanium-to-oscal task produces the `results` attribute of an `assessment-results` object.
 
```

Comment:
I would say OSCAL objects *may* be extremely large.

Reply:
done